cicd-scripts/run-prow-e2e.sh

```diff
@@ -39,3 +39,10 @@ export OPTIONS_MANAGED_PASSWORD=$(yq e '.password' $SHARED_DIR/managed-1.json)
 
 echo -e "\nRunning Search-e2e tests in ${TEST_MODE} test mode. Preparing to run e2e tests."
 ./start-tests.sh
+
+echo "TODO: Uploading test results to AWS S3 bucket."
```

Comment:
Is it possible to use a line like you have in build/upload-to-s3?

`aws s3 sync ./results s3://search-e2e-results/prow-${PROW_BUILD_ID}`

Reply:
For uploading to s3, this will be accommodated within a separate PR. For now, the main focus was to get the tests running within the build; however, there are still some issues we need to resolve for uploading the results.

Reply:
sounds good